pkg/gardenlet/controller/controllerinstallation/controllerinstallation_control.go

```diff
@@ -216,14 +216,25 @@ func (c *defaultControllerInstallationControl) reconcile(controllerInstallation
 		}
 	}
 
+	gardenClusterIdentity, err := getClusterIdentity(ctx, gardenClient.Client(), nil)
+	if err != nil {
+		return fmt.Errorf("failed to retrieve garden's cluster-identity: %s", err)
+	}
+	seedClusterIdentity, err := getClusterIdentity(ctx, seedClient.Client(), seed.Status.ClusterIdentity)
+	if err != nil {
+		return fmt.Errorf("failed to retrieve cluster-identity of seed '%s': %s", seed.Name, err)
+	}
+
 	// Mix-in some standard values for garden and seed.
 	gardenerValues := map[string]interface{}{
 		"gardener": map[string]interface{}{
 			"garden": map[string]interface{}{
-				"identity": c.gardenNamespace.UID,
+				"identity":        c.gardenNamespace.UID,
```

Comment:
Can you add a comment that this field is deprecated and needs to be removed in a future version?

Reply:
done, please check